sourcemash/items/models.py

```diff
@@ -7,6 +7,7 @@ class Item(db.Model):
     last_updated = db.Column(db.DateTime)
     author = db.Column(db.String(120))
     summary = db.Column(db.String(500))
+    text = db.Column(db.Text)
```

Comment:
Are there any length requirements on this that we should keep in mind? Will this need to make us worry about database row issues?